adventure.py

```diff
@@ -0,0 +1,47 @@
+
+def start():
+    print "You are in a house"
+    print "There is a door in front of you."
+    print "\tType \"Open door"
+    choice=raw_input("Enter choice : ")    

```

Comment:
Choices should be printed exactly. Like it is printed "Type Open Door", but the condition below checks for "open"